### ISSUE TITLE
PSQL_TRUST_LOCALNET order incorrect

### ISSUE THREAD

**Issue by lgfausak** [sameersbn/docker-postgresql#14]
The addition to support PSQL_TRUST_LOCALNET needs to set the samenet pg_hba.conf file BEFORE the host all 0.0.0.0 line.  I swapped the order, the new order looks like this:

```
if [ ${PSQL_TRUST_LOCALNET} = "true" ]; then
  echo "Enabling trust samenet in pg_hba.conf..."
  cat >> ${PG_CONFDIR}/pg_hba.conf <<EOF
host    all             all             samenet                 trust
EOF
fi

# allow remote connections to postgresql database
cat >> ${PG_CONFDIR}/pg_hba.conf <<EOF
host    all             all             0.0.0.0/0               md5
EOF
```

Note the PSQL_TRUST_LOCALNET comes first.  Otherwise, the second one masks the first one.

I added this code to my pull request for uid/gid postgres/postgres 600/600, if merged, this will come along with it.

**Comment by sameersbn** [comment]
@lgfausak understood, please send a separate PR for this fix.

**Comment by lgfausak** [comment]
sent

**Comment by sameersbn** [comment]
resolved in commit fec631fe5a857465bdb0c0e83028f0ebc16997d0
